applicationinsights/boot-cljsjs-checksums.edn

```diff
@@ -0,0 +1,2 @@
+{"cljsjs/production/applicationinsights.min.inc.js"
```

Comment:
Hmm, this file should include both files. Run `boot package target` and check that `cljsjs` folder really has the both files, if not, sift move patterns might not be moving the files correctly.

Reply:
It works just fine on my box, but a colleague of mine is running into an issue on Windows with that sift-command he says

This is how it looks on my box:
```
.
├── common
│   └── applicationinsights.ext.js
├── development
│   ├── applicationinsights.inc.js
│   └── applicationinsights.inc.js.map
└── production
    ├── applicationinsights.min.inc.js
    └── applicationinsights.min.inc.js.map

3 directories, 5 files
```

I'll touch back when I know more about that error he's bumping into

Reply:
Even without the `unzip` and the `sift move` tasks it doesn't run properly on windows. But he's not able to package and install other libraries in `cljsjs` either, so I'm thinking it's more of a problem with his box.
Will this be any different when it's pulled in as a dependency?